test/ffd_gpio/src/app_conf.h

```diff
@@ -15,10 +15,10 @@
 #define I2C_SLAVE_CORE_MASK  (1 << 3)
 #define I2C_SLAVE_ADDR       0x7A
 
-#define appconfINTENT_I2C_OUTPUT_DEVICE_ADDR  I2C_SLAVE_ADDR
+#define appconfINTENT_I2C_MASTER_DEVICE_ADDR  I2C_SLAVE_ADDR
```

Comment:
This line caught my attention.  It sounds counter-intuitive given the symbol names.  Is the intention to set the I2C Master Device Address to the I2C Slave Address?

Reply:
I checked that define `appconfINTENT_I2C_OUTPUT_DEVICE_ADDR`   is not used in that example, so I removed it. The unusual combination between master and slave was caused by search/replace action on all the files.